internal/server/data/migrations.go

```diff
@@ -695,3 +697,41 @@ ALTER TABLE provider_users ADD CONSTRAINT
 		},
 	}
 }
+
+func removeDotFromDestinationName() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-10-04T11:44",
+		Migrate: func(tx migrator.DB) error {
+			type idName struct {
+				id   uid.ID
+				name string
+			}
+
+			rows, err := tx.Query(`SELECT id, name FROM destinations WHERE name LIKE '%.%'`)
```

Comment:
I realize now I probably could have done this in a single update statement with the postgres `replace` function (https://www.postgresql.org/docs/current/functions-string.html), but I think this is fine. I'll try to remember that option for next time.